tests/tt_metal/tt_metal/unit_tests_common/gtest_smoke/test_basic_pipeline.cpp

```diff
@@ -0,0 +1,351 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
comment w/ a purpose.  what does this test?  why?

Reply:
comments have been added to describe test purpose